modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -2599,29 +2626,58 @@ public void testDiagnosticPageLocksTracker() throws Exception {
 
         ignite.cluster().active(true);
 
-        String dir = U.defaultWorkDirectory() + "/diagnostic/";
-
         assertEquals(EXIT_CODE_OK, execute("--diagnostic"));
         assertEquals(EXIT_CODE_OK, execute("--diagnostic", "help"));
-        assertEquals(EXIT_CODE_OK, execute("--diagnostic", "pageLocks", "help"));
 
         assertEquals(EXIT_CODE_OK, execute("--diagnostic", "pageLocks", "dump"));
+
+        checkNumberFiles(defaultDiagnosticDir, 1);
+
         assertEquals(EXIT_CODE_OK, execute("--diagnostic", "pageLocks", "dump_log"));
-        assertEquals(EXIT_CODE_OK, execute("--diagnostic", "pageLocks", "dump", dir));
+        assertEquals(EXIT_CODE_OK, execute("--diagnostic", "pageLocks", "dump", "--path",
+            customDiagnosticDir.getAbsolutePath()));
+
+        checkNumberFiles(customDiagnosticDir, 1);
```

Comment:
Why is there only one diagnostic file? It's not clear. Please add comments for all method invocation.